skbio/metadata/intersection.pyx

```diff
@@ -0,0 +1,513 @@
+"""
```

Comment:
@gregcaporaso @jairideout @ebolyen , what do you guys think about making the IntervalTree data structure publically accessible?